e2e/helpers/helpers.ts

```diff
@@ -153,4 +165,51 @@ export class HelpFunctions {
     await this.sessionTransferInput.type(number);
     await this.sessionCompleteTransferButton.click();
   }
+
+  async subsribeToContact(userAuthId: string) {
+    await this.subscriptionInput.click({ clickCount: 3 });
+    await this.page.keyboard.press('Backspace');
+    await this.subscriptionInput.type(`sip:${userAuthId}@voipgrid.nl`);
```

Comment:
Here we can use the REALM from the .env.
```suggestion
    await this.subscriptionInput.type(`sip:${userAuthId}@${process.env.REALM}`);
```
Nicer, if we ever test against another realm :)